src/classes/Util.ts

```diff
@@ -39,4 +39,20 @@ export class Util extends BaseClass {
 		const regex = /^[0-9a-f]{32}$/i;
 		return regex.test(uuid);
 	}
+
+	/**
+	 * Get player's name from their UUID
+	 */
+	public async getUsername(uuid: string) {
+		try {
+			const data = await petitio(`https://api.mojang.com/user/profile/${uuid}`).send();
+			const json = (await data.json()) as GetUsernameResponse;
+
+			if (json.error) throw new HypixelTSError('GET_USERNAME_ERROR', json.error, data.statusCode);
```

Comment:
You need to add the error message, an error will be thrown with the current implementation, check this file and add the messages: https://github.com/hypixelts/hypixel.ts/blob/3890323571ca8be3a99c013b067476edddf0357c/src/errors/Messages.ts

Reply:
fixed in a7c0b62e